src/components/common/SvgIcon.vue

```diff
@@ -0,0 +1,42 @@
+<template>
```

Comment:
These SVGIcon + SVGSprite components look good! I'm wondering if we could leverage fontawesome's library management to offload some of the work/complexity? 

I was personally curious and did a quick google search leading me to [a ts example](https://codesandbox.io/s/fa-list-old-style-ts-k3zhc?file=/src/index.ts) via https://github.com/FortAwesome/vue-fontawesome/issues/274#issuecomment-798897059. Would something like that help us or does it make things more complicated? ⚙️⚙️⚙️

Reply:
Hey @bzabos!  This is a great solution. Unfortunately, it works with only a single path within an svg. Many of our custom SVG icons have multiple paths. I will check with Godspower and Amen if it is possible to combine them into a single path. If so then we can use this approach, I really liked this approach better.

Reply:
@asalami Any thoughts on the possibility to combine multiple paths in an svg to a single path ?  For example: In the screenshot we see two paths under the svg. I need just one path, is this possible? 
<svg> 
<path d=""> 
</svg>
<img width="1227" alt="Screen Shot 2021-07-11 at 1 19 09 PM" src="https://user-images.githubusercontent.com/23651142/125208967-99292580-e24a-11eb-8332-83fcb829b987.png">

Reply:
Fixed